api/v1/types/model_types.go

```diff
@@ -50,10 +50,12 @@ type StatsResponse struct {
 	Stats  map[string]ExternalStatBundle `json:"stats"`
 }
 
-// An ExternalEntityListEntry represents the latest CPU and Memory usage of a model entity.
+// An ExternalEntityListEntry represents the latest CPU, Memory & network usage of a model entity.
 // A model entity can be a Pod, a Container, a Namespace or a Node.
 type ExternalEntityListEntry struct {
 	Name     string `json:"name"`
 	CPUUsage uint64 `json:"cpuUsage"`
 	MemUsage uint64 `json:"memUsage"`
+	RxBytes  uint64 `json:"rx_bytes"`
```

Comment:
So are you saying it shouldn't be returned by API queries, only passed to sinks? I'd prefer it to be exposed via the REST API & that means using the model, doesn't it?